index.html

```diff
@@ -1,12 +1,56 @@
 <!DOCTYPE html>
   <html lang="en">
-  <head>
-     <meta charset="UTF-8">
-     <title>Angular ToDo App</title>
-     <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.5.2/angular.min.js"></script>
-     <script src="app.js"></script>
-  </head>
-  <body>
-    <h1>To Do!</h1>
-  </body>
-</html>
+    <head>
+      <meta charset="UTF-8">
+      <title>Angular ToDo App</title>
+      <link rel="stylesheet" href="bootstrap.css"/>
+      <script src="https://cdnjs.cloudflare.com/ajax/libs/angular.js/2.0.0-beta.11/Rx.min.js"></script>
+      <script src="data.js"></script>
+      <script src="app.js"></script>
+    </head>
+    <body class=container data-ng-app="todo">
+      <header class="jumbotron">
+        <h1>So Much to Do!</h1>
+      </header>
+
+      <main data-ng-controller="ToDoListController as vm">
```

Comment:
You need to make sure that the controller name in this file (`ToDoListController`) matches the name in your actual controller (`todolistController`) in `app.js`. I'm guessing that's the source of your problems here.